elements/pfe-textinput/src/pfe-textinput.scss

```
@@ -0,0 +1,56 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: textinput;
+
+$LOCAL-VARIABLES: (
+  BackgroundColor: pfe-var(surface--lightest),
+  BackgroundColor--disabled: pfe-var(surface--base), // needs to be #f0f0f0,
```

surface-base is #f0f0f0 in master
```suggestion
  BackgroundColor--disabled: pfe-var(surface--base),
```